cirq/google/gate_sets.py

```diff
@@ -17,17 +17,11 @@
 from cirq import ops, protocols
 from cirq.google import op_serializer, op_deserializer, serializable_gate_set
 from cirq.google.common_serializers import (
-    SINGLE_QUBIT_SERIALIZERS,
-    SINGLE_QUBIT_DESERIALIZERS,
-    SINGLE_QUBIT_HALF_PI_SERIALIZERS,
-    SINGLE_QUBIT_HALF_PI_DESERIALIZERS,
-    MEASUREMENT_SERIALIZER,
-    MEASUREMENT_DESERIALIZER,
-    SYC_SERIALIZER,
-    SYC_DESERIALIZER,
-    SQRT_ISWAP_SERIALIZERS,
-    SQRT_ISWAP_DESERIALIZERS,
-)
+    SINGLE_QUBIT_SERIALIZERS, SINGLE_QUBIT_DESERIALIZERS,
+    SINGLE_QUBIT_HALF_PI_SERIALIZERS, SINGLE_QUBIT_HALF_PI_DESERIALIZERS,
+    MEASUREMENT_SERIALIZER, MEASUREMENT_DESERIALIZER, SYC_SERIALIZER,
+    SYC_DESERIALIZER, SQRT_ISWAP_SERIALIZERS, SQRT_ISWAP_DESERIALIZERS,
+    WAIT_GATE_SERIALIZER, WAIT_GATE_DESERIALIZER)
```

Comment:
Add trailing comma so that the formatting remains the same?

Reply:
Done.